pkgs/by-name/al/alacarte/package.nix

```diff
@@ -0,0 +1,70 @@
+{
+  lib,
+  fetchFromGitLab,
+  python3,
+  autoconf,
+  automake,
+  gettext,
+  pkg-config,
+  libxslt,
+  gobject-introspection,
+  wrapGAppsHook,
```

Comment:
This one was renamed in #307077

Reply:
Yeah I was also wondering why there's `wrapGAppsHook4` and no `wrapGAppsHook3` for GTK3. Glad to know it was renamed